lib/index.js

```diff
@@ -162,6 +165,76 @@ export default {
     return provider
   },
 
+  provideHyperclick(): HyperclickProvider {
+    const provider = {
+      grammarScopes: ['source.js', 'source.js.jsx'],
+      getSuggestionForWord: async (textEditor: TextEditor, text: string, range: Range): Promise<?HyperclickSuggestion> => {
+        const filePath = textEditor.getPath()
+        const rootScope = textEditor.getRootScopeDescriptor().scopes[0]
+        console.log(filePath, rootScope, !this.onlyIfAppropriate)
+        if (
```

Comment:
I did not look up the hyperclick code but providing the `grammarScope` in line 170 should be enough, right? If so, remove the complete `if`.